taipy/config/section.py

```diff
@@ -18,9 +18,30 @@
 
 
 class Section:
-    """A Section as a consistent part of the Config.
+    """An abstract class representing a subdivision of the configuration class `Config^`.
+
+    Each Section implementation class defines a semantically consistent set of settings
```

Comment:
I don't understand this.

"The role of the subclasses of `Section^` is to define..."?